evo/tools/tf_cache.py

```diff
@@ -172,10 +161,25 @@ def get_trajectory(self, reader: Rosbag1Reader,
             raise TfCacheException("Could not load trajectory: " + str(e))
         # rosbags Reader start_time is in nanoseconds.
         start_time = rospy.Time.from_sec(reader.start_time * 1e-9)
-        return self.lookup_trajectory(parent, child, start_time=start_time,
-                                      end_time=latest_time)
+        if timestamps is None:
+            timestamps = []
+            # Static TF have zero timestamp in the buffer, which will be lower
+            # than the bag start time. Looking up a static TF is a valid request,
+            # so this should be possible.
+            if latest_time < start_time:
```

Comment:
it might be that the latest time is before the start time in case we used the static tf transform....now I think I understand the situation you were referring to.

This should keep the old behaviour since the `get_trajectory` is the interface that is used externally.